src/utils/helper-functions/guards.ts

```diff
@@ -31,3 +31,15 @@ export function isClassDeclaration(
 ): node is TSESTree.ClassDeclaration {
   return node && node.type === AST_NODE_TYPES.ClassDeclaration
 }
+
+export function isImportDeclaration(
+  node: TSESTree.Node,
+): node is TSESTree.ImportDeclaration {
+  return node.type === 'ImportDeclaration'
```

Comment:
We should also check if `node` is `undefined`, this way it behaves like the other guards